src/WalkingModule/app/robots/iCubGenova09/dcm_walking/common/plannerParams.ini

```diff
@@ -6,7 +6,7 @@ unicycleGain            10.0
 referencePosition       (0.1 0.0)
 timeWeight              2.5
 positionWeight          1.0
-slowWhenTurningGain     5.0
+slowWhenTurningGain     2.5
```

Comment:
This parameter should be increased again as the robot falls down when turning due to the high speed. We could try with `4` if we want to take a higher velocity with respect to the previous value. But the new value should be tested before merging the PR. 

cc @S-Dafarra

Reply:
We won't be able to test the new gain for a while. Should we revert to 5.0 and merge the PR, leaving for later the tuning of this gain?